processor/src/main/java/com/linecorp/decaton/processor/DeferredCompletion.java

```diff
@@ -35,4 +36,16 @@
     default <T> CompletableFuture<T> completeWith(CompletableFuture<T> future) {
         return future.whenComplete((r, e) -> complete());
     }
+
+    /**
+     * Associate completion of this completion to the given {@link Completion}.
+     * By calling this method for the completion X with the argument of A,
+     * * when A completes, X completes
+     * * when X times out, A times out
+     *
+     * Always prefer to use this method instead of {@link #completeWith(CompletableFuture)} when possible.
```

Comment:
> Always prefer to use this method instead of {@link #completeWith(CompletableFuture)} when possible.

I think users (also me :) ) may wonder why is that.
Could you elaborate why in the comment, or just delete the sentence?